Bitbucket.Authentication/Authentication.cs

```diff
@@ -87,25 +87,40 @@ public override void DeleteCredentials(TargetUri targetUri, string username)
         {
             BaseSecureStore.ValidateTargetUri(targetUri);
 
-            Trace.WriteLine("BitbucketAuthentication::DeleteCredentials");
+            Trace.WriteLine($"BitbucketAuthentication::DeleteCredentials for {targetUri.ActualUri}");
```

Comment:
Since `Trace.WriteLine` will print the method name, regardless - would it be more informational or more terse to say `Trace.WriteLine($"Deleting BitBucket credentials for {targetUri.ActualUri}.")` ?